eugl/gqa/tasks.py

```diff
@@ -4,38 +4,53 @@
 QGA Workflow
 -------------
 
-Workflow settings can be configured in `gqa.cfg` file.
-
+Workflow settings can be configured in `luigi.cfg` file.
 """
+
 # pylint: disable=missing-docstring,no-init,too-many-function-args
 # pylint: disable=too-many-locals
 
 from __future__ import print_function
 
+import math
+import re
 import logging
 import os
-from os.path import join as pjoin, dirname, basename, exists, isdir
+from os.path import join as pjoin, dirname, basename, exists, isdir, splitext, abspath
 import glob
 from pkg_resources import resource_filename
 import shutil
 import traceback
 import argparse
+from datetime import datetime
+from dateutil.tz import tzutc
```

Comment:
Nitpick: the utc timezone is also available in datetime.timezone(.utc); if you use this it removes the dependency on dateutil from this file.

Reply:
Done.